### ISSUE TITLE
ReaktoroShared install target not exactly implemeted as ReaktoroStatic install target

### ISSUE THREAD

**Issue by allanleal** [reaktoro/reaktoro#21]
This is what we have for install target of ReaktoroShared
~~~
    install(TARGETS ReaktoroShared
        ARCHIVE
            DESTINATION lib
            COMPONENT libraries
        LIBRARY
            DESTINATION lib
            COMPONENT libraries
        RUNTIME
            DESTINATION bin
            COMPONENT libraries
    )
~~~

and this for ReaktoroStatic:

~~~
install(TARGETS ReaktoroStatic DESTINATION "lib" COMPONENT libraries)
~~~

Shouldn't the first be similar to the second?

**Comment by allanleal** [comment]
Also, I think we should use GNUInstallDirs variables instead of `lib`. @tadeu , just let me know your thoughts and I can proceed with this.

**Comment by tadeu** [comment]
@allanleal Yes, I think they could be the same, it would be more consistent (although there should be not difference in the final installed tree).

> Also, I think we should use GNUInstallDirs variables instead of lib. @tadeu , just let me know your thoughts and I can proceed with this.

I think it's ok to use it, as long as it still allows user customization of `CMAKE_INSTALL_PREFIX` (and respected `lib` and `bin` convention inside it), as this is needed in order to build a relocatable package for packages systems such as `conda`.

**Comment by allanleal** [comment]
This has been fixed with PR #32
